runtime/common/src/parachains.rs

```diff
@@ -115,6 +128,24 @@ impl<AccountId, T: Currency<AccountId>> ParachainCurrency<AccountId> for T where
 
 		Ok(())
 	}
+
+	fn transfer_in(
+		source: &AccountId,
+		dest: ParaId,
+		amount: Balance,
+		existence_requirement: ExistenceRequirement,
+	) -> DispatchResult {
+		T::transfer(source, &dest.into_account(), amount.into(), existence_requirement)
```

Comment:
So this is transferring funds into the parachain (treating it as an account).

Wouldn't we want a static existence requirement for that?